.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+
+### Description
+
+Write here
+
+### Check List
+
+- [ ] All test passed
+- [ ] Added test to ensure to fix/ensure properly.
```

Comment:
You are correct, sir. I just copied and pasted from somewhere... What would you suggest instead?